html/components/team-time-tab.js

```diff
@@ -145,6 +145,8 @@ function createMetaControlTable() {
 
 	var confirmedButton = toggleButton("ScoreBoard.OfficialScore", "Official Score", "Unofficial Score");
 	confirmedButton.appendTo(periodEndTd);
+	var statsLockedButton = toggleButton("ScoreBoard.StatsLocked", "Stats Locked", "Writing Stats");
```

Comment:
As per the issue, this should not involve operator action.

Reply:
Writing does not require operator action - this setting is off by default. If it's never touched and the only inputs between final score and Start New Game are corrections, then the Game data file will contain exactly the expected data.
This button merely adds the option to block (accidental) writes once the NSOs finished their cross-checks, if the next game isn't started right away..

Reply:
The button doesn't lock stats though, they can still be changed - they'll just not get written out to one particular way of getting stats out. I think this would cause more confusion than anything.

Reply:
>one particular way of getting stats out

The one we advertise as the default way.
But I'm open for suggestions as to a better label.

Reply:
I'd suggest not having it, among other things it's not in the videos I just recorded.

The original code missed score updates after the last jam ended. The current code misses paperwork updates after the score is final. What I propose won't miss anything, but if someone messes around might lose data through an overwrite. Let's keep it simple for now, and only add a button if we really need it. A game data manager can always download the data once it's final, and they're likely in a better position to judge that than the SBO (who has probably disappeared off by that stage).

Reply:
> among other things it's not in the videos I just recorded

Alright. So let's stop the bugfixing then.

Reply:
Yeah, best to avoid non-critical UI changes until the release.

Reply:
> best to avoid non-critical UI changes

So I can expect you to withdraw #391 ?

Reply:
No, that's still a bugfix that should go in one way or the other.